AmplifyPlugins/Logging/Sources/AWSCloudWatchLoggingPlugin/Configuration/AWSCloudWatchLoggingPluginConfiguration.swift

```diff
@@ -0,0 +1,105 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Amplify
+
+public struct AWSCloudWatchLoggingPluginConfiguration: Codable {
+    public init(
+        logGroupName: String,
+        region: String,
+        enable: Bool = true,
+        localStoreMaxSizeInMB: Int = 5,
+        flushIntervalInSeconds: Int = 60,
+        defaultRemoteConfiguration: DefaultRemoteConfiguration? = nil,
+        loggingConstraints: LoggingConstraints = LoggingConstraints()
+     ) {
+         self.logGroupName = logGroupName
+         self.region = region
+         self.enable = enable
+         self.localStoreMaxSizeInMB = localStoreMaxSizeInMB
+         self.flushIntervalInSeconds = flushIntervalInSeconds
+         self.defaultRemoteConfiguration = defaultRemoteConfiguration
+         self.loggingConstraints = loggingConstraints
+    }
+
+   public let enable: Bool
+   public let logGroupName: String
+   public let region: String
+   public let localStoreMaxSizeInMB: Int
+   public let flushIntervalInSeconds: Int
+   public let defaultRemoteConfiguration: DefaultRemoteConfiguration?
+   public let loggingConstraints: LoggingConstraints
+}
+
+import Amplify
+import AWSPluginsCore
```

Comment:
nit: import statements at top of the file

Reply:
imports statements removed.